Tests/test_compartments.py

```diff
@@ -0,0 +1,21 @@
+from unittest import TestCase
+from biocrnpyler import Species, Compartment
+import pytest
+
+
+class TestSpecies(TestCase):
+
+    def test_compartment_initialization(self):
```

Comment:
Probably, it's the best not to mix unittest and pytest packages. You can just write a pytest:

```python
def test_compartment_initialization():
    # tests naming convention repr without species type or attributes
      s_name = 'test_species'
      c_name = ''test_compartment''
    species = Species(name=s_name)
    compartment = Compartment(name=c_name)
    species.compartment = compartment
    assert compartment.name == c_name
    assert species.compartment.name == compartment.name
```

Reply:
Could you explain why this is not a good idea and also why do we have the two different packages? I like the unittest framework better because it kinda seems that it stores the results nicely in its classes..but not sure if that's an advantage or how it could be an advantage...

Reply:
At other places in our testing, we have similar mixing of unittest and pytest.

Reply:
In the second unittest case, you use pytest's check for exceptions instead of the unittest built-in one.  Hence, the mixing. It's a minor thing, you can keep it this way. 
That's the unittest's way: 
https://ongspxm.gitlab.io/blog/2016/11/assertraises-testing-for-errors-in-unittest/

Reply:
Yeah, it was using pytest way earlier and I moved some of those functions to be the unittest way. Do we have a preference? In the long run, it might be good to decide on one thing that we slowly migrate towards.

Reply:
My hidden agenda is to migrate to pytest eventually, e.g. I write all my new tests in pytest. But feel free to use either for now.

Reply:
OK then I will move to pytest. Although with unittest there's the `setUp` function that you can create inside the class that's helpful at times (not sure if it's possible with pytest as well).